pages/task/read/routers/confirm.js

```diff
@@ -32,7 +33,7 @@ module.exports = () => {
     const params = { status: stepId };
 
     if (commentRequired(stepId)) {
-      params.comment = formValues[`${stepId}-reason`];
+      params.comment = formValues.reason;
```

Comment:
if we're always allowing a comment now, do we still want this conditional?

Reply:
Good catch, that cleans this section up a lot.